package.json

```diff
@@ -861,6 +861,11 @@
           "default": true,
           "description": "Automatically reveal the terminal when executing a command or task"
         },
+        "sweetpad.system.customXcodeWorkspaceParser": {
```

Comment:
I’ve renamed the config to `sweetpad.system.customXcodeWorkspaceParser`, I hope you’re OK with the change!